apps/watch/src/components/Videos/Videos.tsx

```diff
@@ -22,29 +26,18 @@ export const GET_VIDEOS = gql`
   }
 `
 
-interface VideosProps {
-  filter?: VideosFilter
-  layout?: 'grid' | 'carousel'
-  variant?: 'small' | 'large'
-  limit?: number
-  showLoadMore?: boolean
-}
+export const limit = 20
 
 function isAtEnd(count: number, limit: number, previousCount: number): boolean {
   if (count === previousCount) return true
   return count % limit !== 0
 }
 
-export function Videos({
-  layout = 'grid',
-  filter = {},
-  variant = 'large',
-  limit = 8,
-  showLoadMore = true
-}: VideosProps): ReactElement {
+export function Videos(): ReactElement {
```

Comment:
To match the other pages
```suggestion
export function AllVideosPage(): ReactElement {
```

Reply:
Went with VideosPage since that is the url route